config/routes.js

```diff
@@ -32,9 +32,12 @@ module.exports.routes = {
   *                                                                          *
   ***************************************************************************/
 
-  '/': {
-    view: 'homepage'
-  }
+  '/': '/api/',
+  '/api': 'ApiController',
+  '/api/v1': 'v1/ApiController.index',
+  '/api/v1/user': 'v1/UserController',
```

Comment:
Perhaps this is what's triggering the route binding to `v1/user.index`? Noted above.